lib/generators/sorcery/templates/initializer.rb

```diff
@@ -250,6 +250,12 @@
     #
     # user.subclasses_inherit_config =
 
+    # Set token randomness.
+    # The length of the result string is about 4/3 of `token_randomness`.
+    # Default: `15`
+    #
+    # config.token_randomness =
```

Comment:
s/`config.token_randomness =`/`user.token_randomness =`/

@mtsmfm  Could you actually check that not only the test code but also this setting works?